File: Java Programming/Pratice of Java Programming/program4.java
/*
Q4. Accept number from user and display below pattern.
Input : 4
Output : # 1 * # 2 * # 3 * # 4 *
*/
import java.util.Scanner;
class Logic
{
void Pattern(int iNo)
{
    int i =0;
    
    for ( i = 1; i <= iNo; i++)
    {
        System.out.print("#\t"+i+"\t*\t");
    }
    
}
}
class program4 
{
    public static void main(String[] args) 
    {
        Scanner sobj=new Scanner(System.in);
        int iValue = 0;

        System.out.println("Enter number of elements: ");
        iValue=sobj.nextInt();

        Logic obj=new Logic();

        obj.Pattern(iValue);

        sobj.close();
    }
    
}

File: C++ Programming/Competitive Programming/Coding Challenges 2/program4.cpp
/*
Q. Accept number of rows and number of columns from user and display below
pattern.
Input : iRow = 4 iCol = 5
Output :4 4 4 4 4
        3 3 3 3 3
        2 2 2 2 2
        1 1 1 1 1
*/

#include<iostream>
using namespace std;

void Pattern(int iRow, int iCol)
{
    int i=0;
    int j=0;

    for ( i = 1; i <= iRow; i++)
    {
        for ( j = 1; j <=iCol; j++)
        {
            cout<<iRow-i+1<<"\t";
        }
        cout<<"\n";
    }
}


int main()
{
    int iValue1 = 0, iValue2 = 0;

    cout<<"Enter Number of Rows : ";
    cin>>iValue1;

    cout<<"Enter Number of Columns : ";
    cin>>iValue2;

    Pattern(iValue1, iValue2);

    return 0;
}